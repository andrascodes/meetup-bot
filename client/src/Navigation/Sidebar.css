.Sidebar {
  z-index: 1000;
  position: fixed;
  left: 160px;
  width: 160px;
  height: 100%;
  margin-left: -160px;
  overflow-y: auto;
  background: #464646;
}

.Sidebar-Links {
    position: absolute;
    top: 0;
    width: 160px;
    margin: 50px 0 0 0;
    padding: 0;
    list-style: none;
    text-align: left;
}

.Sidebar-Links li {
    text-indent: 20px;
    line-height: 40px;
}

.Link {
    display: block;
    text-decoration: none;
    color: #c5c5c5;
}

.Link:hover {
    text-decoration: none;
    color: #fff;
    background: rgba(255,255,255,0.2);
}

.Link:active,
.Link:focus {
    text-decoration: none;
    color: #fff;
}