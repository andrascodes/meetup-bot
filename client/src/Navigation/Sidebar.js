import React from 'react'
import { Link } from 'react-router-dom'

import './Sidebar.css'

export const Sidebar = props => (
  <div className="Sidebar">
    <ul className="Sidebar-Links">
      <li>
        <Link className="Link" to="/">Dashboard</Link>
      </li>
      <li>
        <Link className="Link" to="/conversations">Conversations</Link>
      </li>
    </ul>
  </div>
)