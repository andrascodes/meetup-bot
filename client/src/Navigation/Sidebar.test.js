import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import { Sidebar } from './Sidebar'

describe('Sidebar', () => {

  test('renders correctly', () => {
    let tree = renderer.create(
      <MemoryRouter>
        <Sidebar />
      </MemoryRouter>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

})