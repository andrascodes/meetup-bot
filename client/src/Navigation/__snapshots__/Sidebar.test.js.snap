exports[`Sidebar renders correctly 1`] = `
<div
  className="Sidebar">
  <ul
    className="Sidebar-Links">
    <li>
      <a
        className="Link"
        href="/"
        onClick={[Function]}>
        Dashboard
      </a>
    </li>
    <li>
      <a
        className="Link"
        href="/conversations"
        onClick={[Function]}>
        Conversations
      </a>
    </li>
  </ul>
</div>
`;
